packages/docs/docs/tutorial/integrate-sdk/React.mdx

```diff
@@ -1,3 +1,8 @@
+---
+sidebar_label: React
+---
+
+import Step from '@site/src/components/Step';
```

Comment:
Does it have to start with `@site/src`? Would it be possible to use `@/components`?

Reply:
or `@components/`?